src/shared/utils.ts

```diff
@@ -78,6 +79,9 @@ export const makeFilteredArraySchema = <T extends ZodSchema>(schema: T) =>
 
             if (!parsedItem.success) {
                 logger.warn("Error parsing item", parsedItem.error.format());
+                putMetricData(`custom/SharedUtils-${process.env.STAGE}`, [
```

Comment:
I think it might be better if we passed the namespace through as a variable so that there's a bit more context around where the error is happening